test/node/conversation.js

```diff
@@ -12,56 +12,70 @@ let user;
 let client2;
 let user2;
 let conversation;
-let timeElapsed;
 describe('Conversation Tests', () => {
     before(async () => {
         client = new Circuit.Client(config.bot1);
         user = await client.logon();
         client2 = new Circuit.Client(config.bot2);
         user2 = await client2.logon();
         conversation = prep.conversation;
-        timeElapsed = Date.now() - conversation.creationTime;
-        // conversation = await client.createConferenceBridge(`${Date.now()}title`);
     });
 
     after(async () => {
         await client.logout();
         await client2.logout();
     });
 
-    it('should get the conversation', async () => {
+    it('should get first page of latest conversations', async () => {
         const res = await client.getConversations();
         assert(res && res.some(conv => conv.convId === conversation.convId));
     });
 
-    it('should get the conversation using AFTER', async () => {
+    // it('should get a number of participants for conversation', async () => {
+    //     const res = await client.getConversations({
+    //         numberOfParticipants: 1
+    //     });
+    //     assert(res && res.some(conv => conv.convId === conversation.convId) && res.every(conv => conv.participants.length <= 1));
+    // });
+
+    it('should get a specified number of conversations', async () => {
+        //random number of conversations to retrieve numberOfConvs 
+        const numberOfConvs = Math.floor(Math.random() * 10) + 1;
```

Comment:
We should just hardcode this to 3 for example. We need to ensure that there are 3 conversations though.

Reply:
done